src/api/Api.js

```diff
@@ -0,0 +1,30 @@
+const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/FN12ot9XO2ctzueymCoY/books';
+export const fetchBook = async () => {
+  const res = await fetch(url);
+  console.log(res.json());
```

Comment:
- [ ] [OPTIONAL] Let's kindly make sure we remove the unwanted piece of code for production 😇

Reply:
Thank you @Amrhub, I will obviously consider your suggestion.